src/Operations.jl

```diff
@@ -77,7 +80,7 @@ end
 function set_maximum_version_registry!(env::EnvCache, pkg::PackageSpec)
     pkgversions = Set{VersionNumber}()
     for path in registered_paths(env, pkg.uuid)
-        pathvers = keys(load_versions(path))
+        pathvers = keys(load_versions(path; include_yanked = true)) # ??
```

Comment:
Probably `false` here.

Reply:
Ok, what does this do? So we can add a test

Reply:
It is for backward compatibility with 0.6 stuff so not sure we need to add a test really. It just sets the version to the maximum one in the registry, only used when no Project file is present.